src/createCycleMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import {run} from '@cycle/xstream-run';
+import actionDriver, { actionListener } from './drivers/actionDriver';
+import stateDriver, { stateListener } from './drivers/stateDriver';
```

Comment:
Honestly, this looks kinda odd and singleton-ish. Maybe we should still init these in `createCycleMiddleware` *and* pass them to the driver, along with `store`?